api/src/main/java/org/commonjava/indy/conf/DefaultIndyConfiguration.java

```diff
@@ -290,4 +299,31 @@ public Properties getSystemPropertyAdditions()
 
         return props;
     }
+    
+    // #!#
+    @ConfigName(value = "remote.ssl.required")
+    public void setSSLEnabled(Boolean sslEnabled) 
```

Comment:
Can we change this field / method to `sslRequired`?

Reply:
Sure , i wad just following java naming conventions for boolean fields ( from what i remembered i must admit )